py2many/inference.py

```diff
@@ -404,6 +404,16 @@ def visit_BinOp(self, node):
                 node.annotation = ast.Name(id="float")
             return node
 
+        # Container multiplication
```

Comment:
some of this is covered by `LEGAL_COMBINATIONS` further below. Perhaps moving the two pieces of code closer together could help the flow.

Also, `1 + 2j` errors out here. Perhaps make it legal while you're here?

Reply:
Moved as requested.  And created https://github.com/adsharma/py2many/issues/379 for the unrelated `complex` set of binops, as tests also need creating for them.

Reply:
I see two instances of this code in the diff. Was it intentional?

Reply:
No. Resulted from multiple rebases.